cosmic-core/server/src/main/java/com/cloud/network/NetworkModelImpl.java

```diff
@@ -1815,6 +1816,14 @@ public boolean isNetworkAvailableInDomain(final long networkId, final long domai
             usedIps.add(NetUtils.ip2Long(ip));
         }
 
+        for (final String ip : excludedIps) {
+            if (requestedIp != null && requestedIp.equals(ip)) {
+                s_logger.warn("Requested ip address " + requestedIp + " is in excluded IPs range" + ((NetworkVO)network).getIpExclusionList());
```

Comment:
A space or `: ` after the word _range_? Else it's going to look like: `Requested ip address 1.1.1.1 is in excluded IPs range1.1.1.1-1.1.1.255`